client/src/singletons/bartop-api.js

```diff
@@ -0,0 +1,3 @@
+import BarTopAPI from '../apis/bartop';
```

Comment:
Out of curiosity, why even add this layer? Why not import from the `apis/` folder directly?

Reply:
I see what you're saying. I think that's fine, I hadn't really thought about it like that and it seemed like it may be overkill. But if that's helping your organization, looks good to me.